.github/workflows/pr_check_load_test.yml

```diff
@@ -43,13 +47,49 @@ jobs:
 
     - name: Azure authentication
       uses: azure/login@v1
+      continue-on-error: false
       with:
-        creds: ${{ secrets.AZURE_CREDENTIALS }}
-  
+        client-id: ${{ secrets.AZURE_CLIENT_ID }}
+        tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+        subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+
+    - name: Generate GUID
+      id: guid
+      run: |
+          if [[ "${{ runner.os }}" == "Windows" ]]; then
+            echo "::set-output name=GUID::$(powershell -Command "[guid]::NewGuid().ToString()")"
+          else
+            echo "::set-output name=GUID::$(uuidgen)"
+          fi
+      shell : bash
```

Comment:
shell should not be required?

Reply:
if [[ "${{ runner.os }}" == "Windows" ]]; then
            echo "::set-output name=GUID::$(powershell -Command "[guid]::NewGuid().ToString()")"
          else
            echo "::set-output name=GUID::$(uuidgen)"
          fi
          
          this is bash script, and windows wont understand, if i dont define shell

Reply:
windows mein default is power shell and this is not power shell script